content/en/dora_metrics/setup/deployments.md

```diff
@@ -116,6 +116,22 @@ There are two requirements for calculating Change Lead Time:
 1. Both the Git repository URL and commit SHA are provided when sending deployment events.
 2. Your repository metadata is being [synchronized to Datadog](#synchronize-repository-metadata-to-datadog).
 
+### Breakdown metrics
+
+DataDog also provides the following breakdown metrics, which represent the different stages since a commit is made until it is deployed.
+
+For computing these metrics, the PR associated with a commit must be identified, if any. A commit is associated with a PR if the commit is first introduced to the target branch when merging that PR.
+
+If a commit does not have an associated PR, only Time to Deploy and Deploy Time are emitted.
+
+- `dora.time_to_pr_ready`: Time from the commit until the PR is ready for review. This metric is only emitted for commits that were made before the PR is ready for review.
+- `dora.review_time`: Time since the PR is ready for review until it receives the last approval. This metric is only emitted for commits that were made before the PR is approved.
```

Comment:
"Time from when the PR is marked ready for review until it receives the last approval. This metric is only available for commits that were made before the PR is approved."